src/app/clusters/scenes/scenes.cpp

```diff
@@ -1275,6 +1305,14 @@ bool emberAfPluginScenesServerParseViewScene(app::CommandHandler * commandObj, c
                     }
                 }
             }
+    #endif
+    #ifdef ZCL_USING_MODE_SELECT_CLUSTER_SERVER
+            if (entry.hasCurrentMode)
+            {
+                emberAfPutInt16uInResp(ZCL_CURRENT_MODE_ATTRIBUTE_ID);
```

Comment:
This is nonsense; attribute ids are 16-bit....  Yes, I know this is copy/pasting other nonsense code in this file, but that just raises the question of why we are doing this if this file is all broken anyway.